Tests/DecodableTests.swift

```diff
@@ -165,7 +165,7 @@ class DecodableTestCase: BaseTestCase {
 
         // When
         do {
-            let _ = try String(json: json)
+            let _ = try String(json: json as AnyObject)
```

Comment:
Compiler doesn't like declaring a dictionary literal as type `AnyObject`. I moved the cast up to the declaration though instead of doing it in the function call.